ballerina-to-openapi/build.gradle

```diff
@@ -38,6 +38,7 @@ dependencies {
         exclude group: "javax.validation", module: "validation-api"
     }
     implementation "io.swagger.core.v3:swagger-models"
+    implementation "io.swagger.parser.v3:swagger-parser-safe-url-resolver:${swaggerParserVersion}"
```

Comment:
How do we use this `parser-safe-url-resolver `, is it using the swagger parser itself or do we need to use it within our implementation separately?

Reply:
This is a dependency of `swagger-parser`, from this latest version. There were `NoClassDefFound` errors without this as a dependency